<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[launch Browser]]>
    </line>
    <line>
      <![CDATA[login ]]>
    </line>
    <line>
      <![CDATA[logout]]>
    </line>
    <line>
      <![CDATA[Browser close]]>
    </line>
  </reporter-output>
  <suite started-at="2023-02-01T21:11:28 IST" name="Default suite" finished-at="2023-02-01T21:11:42 IST" duration-ms="13394">
    <groups>
    </groups>
    <test started-at="2023-02-01T21:11:28 IST" name="Default test" finished-at="2023-02-01T21:11:42 IST" duration-ms="13394">
      <class name="test.test">
        <test-method is-config="true" signature="lb()[pri:0, instance:test.test@7d0b7e3c]" started-at="2023-02-01T21:11:28 IST" name="lb" finished-at="2023-02-01T21:11:38 IST" duration-ms="9570" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[launch Browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- lb -->
        <test-method is-config="true" signature="Login()[pri:0, instance:test.test@7d0b7e3c]" started-at="2023-02-01T21:11:38 IST" name="Login" finished-at="2023-02-01T21:11:39 IST" duration-ms="593" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login ]]>
            </line>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method signature="VerifyLogo()[pri:0, instance:test.test@7d0b7e3c]" started-at="2023-02-01T21:11:39 IST" name="VerifyLogo" finished-at="2023-02-01T21:11:41 IST" duration-ms="1995" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[failed expected [ADD TO CARTy] but found [ADD TO CART]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: failed expected [ADD TO CARTy] but found [ADD TO CART]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at test.test.VerifyLogo(test.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogo -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:test.test@7d0b7e3c]" started-at="2023-02-01T21:11:41 IST" name="logout" finished-at="2023-02-01T21:11:41 IST" duration-ms="658" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyLogo status=FAILURE method=test.VerifyLogo()[pri:0, instance:test.test@7d0b7e3c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="CloseBrowser()[pri:0, instance:test.test@7d0b7e3c]" started-at="2023-02-01T21:11:41 IST" name="CloseBrowser" finished-at="2023-02-01T21:11:42 IST" duration-ms="461" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Browser close]]>
            </line>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- test.test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
